tidy3d/components/lumped_element.py

```diff
@@ -272,28 +378,504 @@ def _sheet_conductance(self):
+        numer = 0
+        denom = 0
+        for a_m, m in zip(a, range(len(a))):
+            numer += a_m * K_tan ** (m)
+        for b_m, m in zip(b, range(len(b))):
+            denom += b_m * K_tan ** (m)
+        # We do not include the scaling factor associated with the cell size, since we will
+        # distribute the network over more than one cell.
+        return numer / denom
+
+    @staticmethod
+    def complex_permittivity(
+        a: tuple[float, ...], b: tuple[float, ...], freqs: np.ndarray, dt: float = None
+    ):
+        """
+        Returns an equivalent complex permittivity of the lumped network over the range of frequencies
+        provided in ``freqs`` using the expression in _`[1]`. The result needs to be combined with a
+        :math:`\\epsilon_\\infty`, e.g., 1 or the existing background medium, before being added to an
+        FDTD simulation.
+
+        Parameters
+        ----------
+        a : tuple[float, ...]
+            Coefficients of the numerator polynomial
+        b : tuple[float, ...]
+            Coefficients of the denominator polynomial.
+        freqs: np.ndarray
+            Frequencies at which to evaluate model.
+        dt : float
+            Time step of the FDTD simulation.
+
+        Returns
+        -------
+        np.ndarray
+            The equivalent frequency-dependent portion of the electric permittivity.
+        """
+
+        # For fitting with a pole-residue model, we provide a convenience function for
+        # converting the complex conductivity to a complex permittivity.
+        sigma = NetworkConversions.complex_conductivity(a, b, freqs, dt)
+        return 1j * sigma / (2 * np.pi * freqs * EPSILON_0)
+
+
+class RLCNetwork(Tidy3dBaseModel):
+    """Class for representing a simple network consisting of a resistor, capacitor, and inductor.
+    Provides additional functionality for representing the network as an equivalent medium.
+
+    Notes
+    -----
+
+        Implementation is based on the equivalent medium introduced by _`[1]`.
+
+        **References**
+
+        .. [1]  J. A. Pereda, F. Alimenti, P. Mezzanotte, L. Roselli and R. Sorrentino, "A new algorithm
+                for the incorporation of arbitrary linear lumped networks into FDTD simulators," IEEE
+                Trans. Microw. Theory Tech., vol. 47, no. 6, pp. 943-949, Jun. 1999.
+    """
+
+    resistance: Optional[pd.PositiveFloat] = pd.Field(
+        None,
+        title="Resistance",
+        description="Resistance value in ohms.",
+        unit=OHM,
+    )
+
+    capacitance: Optional[pd.PositiveFloat] = pd.Field(
+        None,
+        title="Capacitance",
+        description="Capacitance value in farads.",
+        unit=FARAD,
+    )
+
+    inductance: Optional[pd.PositiveFloat] = pd.Field(
+        None,
+        title="Inductance",
+        description="Inductance value in henrys.",
+        unit=HENRY,
+    )
+
+    network_topology: Literal["series", "parallel"] = pd.Field(
```

Comment:
There could be other mixed network topologies as well?

Reply:
No, for the `RLCNetwork` it is either 1-3 elements in series or 1-3 elements in parallel.

Reply:
hmm, that's not what I see on wikipedia:
https://en.wikipedia.org/wiki/RLC_circuit#Other_configurations

Reply:
Oh, I meant that is all we support when using the `RLCNetwork`. I made the `AdmittanceNetwork` to support all other types of networks.